src/System.Net.Http/src/netcore50/project.json

```diff
@@ -16,6 +16,7 @@
         "System.Resources.ResourceManager": "4.0.0",
         "System.Runtime.Extensions": "4.0.10",
         "System.Runtime.WindowsRuntime": "4.0.10",
+        "System.Security.Cryptography.X509Certificates": "4.1.0-rc3-24008-00",
```

Comment:
* In theory, everything works.
* In theory, we run the tests (we had a failure in the ToF tree for UWP because of UTC vs PDT host issues, so I'm pretty sure we run them)

But I don't know that anyone has created a new UWP project, referenced the crypto packages, written a program that used them, and executed it on a phone.  Doing that is definitely on my TODO list, but other higher urgency work keeps slipping in front of it.